modules/gapi/src/backends/cpu/gcpubackend.cpp

```diff
@@ -199,7 +240,12 @@ void cv::gimpl::GCPUExecutable::run(std::vector<InObj>  &&input_objs,
         }
 
         // Now trigger the executable unit
-        k.apply(context);
+        if (k.m_isStateful)
+        {
+            context.m_state = m_nodesToStates[op_info.nh];
```

Comment:
Use `.at` here instead of `[]`. Missing a pre-initialized state for a stateful kernel should be fatal.

Reply:
Thanks a lot!!